### PR TITLE
fix Issue 14396 - compile error std.conv.parse!int with input range (dmd...

### PR DESCRIPTION
**Pull request by aG0aep6G**
...2.067)

https://issues.dlang.org/show_bug.cgi?id=14396

### CODE REVIEW

**Comment by John-Colvin** [comment]
LGTM.

Could have just used std.utf.byCodeUnit for the test,

**Comment by CyberShadow** [comment]
Since this fixes a regression, shouldn't this target the stable branch?

**Comment by aG0aep6G** [comment]
> Since this fixes a regression, shouldn't this target the stable branch?

I don't know. If it should, can I change that for this PR? Or do I close this one and make a new one to stable? Or can you just pull it to stable?

File: std/conv.d [hunk 1]
```diff
@@ -2206,6 +2206,22 @@ Lerr:
...
 }
 
+// Issue 14396
+@safe pure unittest
+{
+    struct StrInputRange
+    {
+        this (string s) { str = s; }
+        char front() const @property { return str[front_index]; }
+        char popFront() @property { return str[front_index++]; }
```
Comment (burner) [inline std/conv.d]: popFront is a function not a property.

Comment (aG0aep6G) [inline std/conv.d]: fixed

**Comment by tramker** [comment]
>I don't know. If it should, can I change that for this PR? Or do I close this one and make a new one to stable? Or can you just pull it to stable?

You must create a new PR.

**Comment by aG0aep6G** [comment]
> You must close this one and create a new PR.

Done: #3183

**Pull request closed without merging**